platform/qt/src/qmapboxgl.cpp

```diff
@@ -806,27 +930,47 @@ void QMapboxGL::setPaintProperty(const QString& layer_, const QString& property,
     }
 }
 
+/*!
+    Returns true when the map is completely rendered, false otherwise. A partially
+    rendered map ranges from nothing rendered at all to only labels missing.
+*/
 bool QMapboxGL::isFullyLoaded() const
 {
     return d_ptr->mapObj->isFullyLoaded();
 }
 
+/*!
+    Pan the map by \a offset in pixels.
+*/
 void QMapboxGL::moveBy(const QPointF &offset)
 {
     d_ptr->mapObj->moveBy(mbgl::ScreenCoordinate { offset.x(), offset.y() });
 }
 
+/*!
+    Scale the map by \a scale_ in the direction of the \a center. This function
```

Comment:
“In the direction of the center” may not be clear to someone unfamiliar with implementing map gestures. Perhaps note that `center` corresponds to the same coordinate before and after calling this method.